### ISSUE TITLE
One research costs almost $4 lol

### ISSUE THREAD

**Issue by elvinagam** [mshumer/ai-researcher#9]

**Comment by pas-mllr** [comment]
Sound like a bargain. How was the quality? Did you run into any rate limitation issues? If so, how did you solve them? If not, what was your research topic / query?

**Comment by elvinagam** [comment]
i don't think this is a bargain lol. Didn't proceed with it. So, won't be able to help with that.
